techniques/android/MASTG-TECH-0131.md

```diff
@@ -43,3 +47,40 @@ BUILD SUCCESSFUL in 6s
 ```
 
 The report was generated in 3 different formats (HTML, JSON and XML) and can be found in the project directory in `app/build/reports`.
+
+In case there are dependencies you want to suppress, either as they are false positives or as they are not included into the APK, but might be needed to build the APK, you can use a suppression file. The following file would exclude all vulnerabilities from the package URLs `pkg:maven/io.grpc/grpc.*` and `pkg:maven/io.netty/netty.*`:
```

Comment:
```suggestion
**Suppressing False Positives:**

If there are dependencies that you want to suppress because they are false positives or are not included in the APK but might be necessary for building the APK, you can use a suppression file. The following `suppression.xml` would exclude all vulnerabilities from the package URLs `pkg:maven/io.grpc/grpc.*` and `pkg:maven/io.netty/netty.*`:
```